CODE_SNIPPETS.md

```diff
@@ -0,0 +1,251 @@
+# Code Snippets
```

Comment:
@sudoku-lord  I am concerned with putting this in the repo until the snippets have been thoroughly tested